File: 27-remove-element/remove-element.ts
function removeElement(nums: number[], val: number): number {
    let ret = 0
    let l = 0
    let n = nums.length
    for (let r = 0; r<n;r++){
        if (nums[r] != val){
            nums[l] = nums[r]
            l++
            ret++
        }
    }
    return ret
};

File: 27-remove-element/remove-element.py
class Solution:
    def removeElement(self, nums: List[int], val: int) -> int:
        ret = 0
        n = len(nums)
        l = 0
        for r in range(n):
            if nums[r] != val:
                nums[l] = nums[r]
                ret+=1
                l+=1
        return ret